manager/apis/app/v1alpha1/datacatalog_types.go

```diff
@@ -18,6 +16,44 @@ const (
 	DeleteFlow DataFlow = "delete"
 )
 
+// S3Connection contains the connection details to an asset which resides in S3
```

Comment:
We are not re-designing various connection structures that are defined in the taxonomy.
Please leave DataStore definition as-is until open REST API server replaces the existing grpc server for catalog connector.

Reply:
Sure will do that. Thanks